internal/function/uast.go

```diff
@@ -155,7 +155,7 @@ func (f UAST) Eval(ctx *sql.Context, row sql.Row) (interface{}, error) {
 			return nil, err
 		}
 
-		lang = lng.(string)
+		lang = strings.ToLower(lng.(string))
```

Comment:
Should we handle this? or should we let LA know that bblfsh doesn't look for the driver properly because of a capitalization issue? It should work with the outcome enry is producing without any manipulation, right?

Reply:
That's assuming bblfsh wants to keep the same names Enry returns. @ajnavarro what do you think?